core/src/main/scala/com/avast/grpc/jsonbridge/GrpcJsonBridgeBase.scala

```diff
@@ -0,0 +1,36 @@
+package com.avast.grpc.jsonbridge
+
+import com.google.protobuf.Message
+import com.google.protobuf.util.JsonFormat
+import io.grpc.Metadata
+import io.grpc.stub.MetadataUtils
+
+import scala.concurrent.Future
+
+trait GrpcJsonBridgeBase[Stub <: io.grpc.stub.AbstractStub[Stub]] {
```

Comment:
I tend to use `abstract class` for `Base` classes.

Reply:
Don't see a reason for that.. :-)